app/jobs/preservation_submission_job.rb

```diff
@@ -6,15 +6,13 @@ def perform(theses)
     results = { total: theses.count, processed: 0, errors: [] }
     theses.each do |thesis|
       Rails.logger.info("Thesis #{thesis.id} is now being prepared for preservation")
-      sip = thesis.submission_information_packages.create
+      sip = thesis.submission_information_packages.create!
       preserve_sip(sip)
       Rails.logger.info("Thesis #{thesis.id} has been sent to preservation")
       results[:processed] += 1
     rescue StandardError, Aws::Errors => e
```

Comment:
Hmmm. Does `create!` failing cause the rescue block to be triggered? If so, the `error` status would still work right? If not, should we update the rescue to also catch the specific  exception being throw so we get a useful `results[:errors]` list?

Reply:
`create!` failing does trigger the `rescue` block. However, the `error` status does not work in this case because an object is not created when `create!` fails, thus there is no SIP of which we can change the `preservation_status`.

Reply:
lol...how many times will you have to explain this to me? I feel like it's at least 4 so far. <sorry!>